examples/jsm/controls/DragControls.js

```diff
@@ -12,12 +12,21 @@ const _raycaster = new Raycaster();
 
 const _pointer = new Vector2();
 const _offset = new Vector3();
+const _diff = new Vector2();
+const _start = new Vector2();
+const _end = new Vector2();
 const _intersection = new Vector3();
 const _worldPosition = new Vector3();
 const _inverseMatrix = new Matrix4();
 
+const _up = new Vector3();
+const _horizen = new Vector3();
+const _lookAt = new Vector3();
+
 class DragControls extends EventDispatcher {
 
+	mode = "translate";
```

Comment:
Please move this code into the constructor.

Reply:
done